homeassistant/components/alarm_control_panel/arlo.py

```diff
@@ -0,0 +1,116 @@
+"""
+This component provides HA alarm_control_panel support for Arlo System.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.arlo/
+"""
+import asyncio
+import logging
+import voluptuous as vol
+
+from homeassistant.components.arlo import (DATA_ARLO, CONF_ATTRIBUTION)
+from homeassistant.components.alarm_control_panel import (AlarmControlPanel,
+                                                          PLATFORM_SCHEMA)
+from homeassistant.const import (ATTR_ATTRIBUTION, STATE_ALARM_ARMED_AWAY,
+                                 STATE_ALARM_ARMED_HOME, STATE_ALARM_DISARMED)
+import homeassistant.helpers.config_validation as cv
```

Comment:
Please keep the import [ordered](https://home-assistant.io/developers/development_guidelines/).